nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexMojo.java

```diff
@@ -167,7 +167,7 @@ private void generateHtml(Set<String> allFailures, Configuration config) {
         html += "<th>Test Name</th>";
         for (int iter = 0; iter < this.executions.size(); iter++) {
             html += "<th>";
-            html += "" + config.seed;
+            html += "" + this.executions.get(iter).getConfiguration().seed;
```

Comment:
Actually, keep it as is, but add above the seed what was already there "Run i" maybe?